lib/px/utils/pxtokenv1.lua

```diff
@@ -55,6 +55,11 @@ function TokenV1:process()
         error({ message = "mobile_sdk_connection_error" })
     end
 
+    if cookie == "3" then
+        self.px_logger.error("mobile sdk pinning error")
```

Comment:
This is something that we should try to avoid (see EN-4382). The log level of a message that indicates an expected flow, should be `info`, or at worst `warning`.
Log level of `error` should indicated something that the application doesn't expect (e.g. collector returns response code > 400)